portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,90 @@
+                <li><a id="linkedin" onmouseover="highlightLinkedIn()" onmouseleave="unhighlightLinkedIn()" href="https://www.linkedin.com/in/anthony-odum-bb1b46145" target="_blank"><i class="fa fa-linkedin"></i></a></li>
+              </ul>
+          </nav>
+
+            <div id="bubblelabel">
+              <h3><span id="aboutlabel">About</span>&nbsp;&bull;&nbsp;</h3>
+              <h3><span id="projectslabel">Projects</span>&nbsp;&bull;&nbsp;</h3>
+              <h3><span id="resumelabel">Resume</span>&nbsp;&bull;&nbsp;</h3>
+              <h3><span id="githublabel">Github</span>&nbsp;&bull;&nbsp;</h3>
+              <h3><span id="linkedinlabel">Linked In</span></h3>
+            </div>
+        </header>
+      </div>
+
+      <div id="abouttab">
+        <div id="aboutbox">
+          <div onclick="transitionAboutToOriginal()" id="exit"><i class="fa fa-window-close"></i></div>
+          <h2>Hi, My name is Anthony and this is my portfolio site.</h2>
+          <h3>While currently focusing in Java, I also have experience in Javascript, TypeScript, HTML and CSS. Links to my projects, Resume, Github and LinkedIn can all be found here. </h3>
+          <div id="illustration"><img src="images/about.gif" alt="smile"></div>
+          <p id="aboutfooter">Thank you for taking the time to view my site and I look forward to hearing from you. Have a nice day!</p>
+        </div>
+      </div>
+
+      <div id="projectstab">
+          <div id="projectsbox">
+            <div onclick="transitionProjectsToOriginal()" id="exit"><i class="fa fa-window-close"></i></div>
+            <div id="spacing"></div>
+            <div class="workbox">
+              <div class="workimage"><img src="images/SudokuPhoto.png" alt=""></div>
+              <div class="workwords">
+                <h2>Sudoku Generator</h2>
+                <p>This program generates a solvable 9x9 Sudoku board. The user is allowed 5 mistakes to try and solve the board.</p>
+              </div>
+            </div>
+            <div class="workbox">
+              <div class="workimage"><img src="images/Website.png" alt=""></div>
+              <div class="workwords">
+                <h2>Portfolio Website</h2>
+                <p>The website you are currently on!</p>
+              </div>
+            </div>
+            <div class="workbox">
+              <div class="workimage"><img src="images/google.png" alt=""></div>
+              <div class="workwords">
+                  <h2>AMP Performance Visualizer</h2>
+                  <p>An API that downloads and caches AMP webpages, retrieves performance metrics from them and returns them in multiple graphs for the user! I worked on this during my first internship at Google.</p>
+              </div>
+            </div>
+          </div>
+        </div>
     </div>
   </body>
-</html>
+</html>
```

Comment:
nit: "No newline at end of file"... idk, it seems more like a convention that is coming down from C (https://thoughtbot.com/blog/no-newline-at-end-of-file) But it is still good just to get rid of the warning in git diff.

Reply:
Where do you find the git diff warnings? I'm confused on the change that was made here.